its_live_monitoring/src/sentinel2.py

```diff
@@ -35,23 +42,55 @@ def raise_for_missing_in_google_cloud(scene_name: str) -> None:  # noqa: D103
     response.raise_for_status()
 
 
+def add_data_coverage_to_item(item: pystac.Item) -> pystac.Item:  # noqa: D103
+    """Adds the amount of the tile covered by valid data as the property 's2:data_coverage'.
+
+    Raises 'requests.HTTPError' if no tile info metadata can be found.
+
+    Args:
+        item: The desired stac item to add data coverage too.
+
+    Returns:
+        item: The stac item with data coverage added.
+    """
+    tile_info_path = item.assets['tileinfo_metadata'].href[5:]
+
+    response = requests.get(f'https://roda.sentinel-hub.com/{tile_info_path}')
+    response.raise_for_status()
+
+    item.properties['s2:data_coverage'] = response.json()['dataCoveragePercentage']
```

Comment:
This is a decimal ratio, but we compare it to an integer percentage. I think we need to multiply it by 100.
```suggestion
    item.properties['s2:data_coverage'] = float(response.json()['dataCoveragePercentage']) * 100
```